_includes/locale/Locale.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ */
+
+  declare(strict_types=1);
+
+  namespace Keyman\Site\com\keyman;
+
+  class Locale {
+    public const DEFAULT_LOCALE = 'en';
+
+    public const CROWDIN_LOCALES = array(
+      'en',
+      'es-ES',
+      'fr-FR'
```

Comment:
These should be minimal BCP-47 tags - i.e. we would use `es-419` for Latin America and Caribbean, `es` for Spain. Similar for French, should be just `fr`, not `fr-FR` (we would use `fr-CA` for example).